README.md

```diff
@@ -40,15 +40,48 @@ Run the following commands from the project's root directory to pull in both fro
 
 #### Database
 
-Run the following commands from the root directory to initialize the database (note: make sure you have a MySQL database and Redis running):
+Sara Alert targets compatibility with MySQL 5.7.
 
-* (optional for local development only, not for docker containers)
-  ```
-  mysql -u root -e "CREATE USER 'disease_trakker'@'localhost'; GRANT ALL PRIVILEGES ON *.* TO 'disease_trakker'@'localhost'; create database disease_trakker_test; create database disease_trakker_development;"
-  ```
-* `bundle exec rake db:drop db:create db:migrate db:setup`
+**macOS Installation Instructions:**
+
+**Migrating from a newer verison down to MySQL 5.7:**
+Before installing MySQL, if newer versions (i.e. version 8 of MySQL) have previoulsy been installed and database have been created in the newer format, these new-format databses will be incompatible with the older 5.7 version. A simple way to install 5.7 after newer versions is to delete the MySQL data directory and let the 5.7 version recreate it in the correct format. Take care to **back up** existing databases before executing the following command, as it will delete them:
+
+```sh
+rm -rf /usr/local/var/mysql # remove the macOS mysql data directory and any existing databases
+```
+
+If SaraAlert had already been installed with a newer version of MySQL, navigate to the project directory and uninstall the `mysql2` gem. It needs to build the correct native extensions for MySQL 5.7.
+
+```sh
+gem uninstall mysql2 # uninstall mysql2
+```
+
+Newer versions of MySQL also start with incompatible options for version 5.7, so if you had a newer version installed edit `/usr/local/etc/my.cnf` to remove the line `'mysqlx-bind-address=127.0.0.1'`.
+
+**Installation:**
+
+Now, install `mysql@5.7` using [`homebrew`](https://brew.sh):
+
+```sh
+`brew install mysql@5.7`
+`brew link mysql@5.7 --force`
```

Comment:
When running this, I got:

```
Linking /usr/local/Cellar/mysql@5.7/5.7.32... 
Error: Could not symlink bin/innochecksum
Target /usr/local/bin/innochecksum
is a symlink belonging to mysql. You can unlink it:
  brew unlink mysql

To force the link and overwrite all conflicting files:
  brew link --overwrite mysql@5.7

To list all files that would be deleted:
  brew link --overwrite --dry-run mysql@5.7
```

I tried `brew link --overwrite mysql@5.7`, then got:

```
Warning: mysql@5.7 is keg-only and must be linked with --force
```

I then tried a `brew link --overwrite mysql@5.7 --force` and it seemed to do the trick.

Reply:
Hmm I think this because I neglected the instruction to uninstall mysql 8